docs/_config.yml

```diff
@@ -39,6 +39,10 @@ scala_version: "2.11"
 # This suffix is appended to the Scala-dependent Maven artifact names
 scala_version_suffix: "_2.11"
 
+# Plain flink-shaded version is needed for e.g. the hive connector.
+# Please update the shaded_version once new flink-shaded is released.
+shaded_version: "7.0"
```

Comment:
isn't the latest one 8.0? https://mvnrepository.com/artifact/org.apache.flink/flink-shaded

Reply:
you are right, I found that the flink master's pom.xml is also used in 8.0 and I have modified it, PTAL.